configure.ac

```diff
@@ -2742,12 +2742,11 @@ oc_bytecode_cflags="$oc_cflags"
 oc_bytecode_cppflags="$oc_cppflags"
 
 oc_native_cflags="$oc_cflags $oc_native_cflags"
-native_cflags="$common_cflags $native_cflags"
 oc_native_cppflags="$oc_cppflags $oc_native_cppflags"
 
 bytecode_cflags="$common_cflags $sharedlib_cflags\
  $PTHREAD_CFLAGS $COMPILER_BYTECODE_CFLAGS"
-native_cflags="$native_cflags $PTHREAD_CFLAGS $COMPILER_NATIVE_CFLAGS"
+native_cflags="$native_cflags $bytecode_cflags"
```

Comment:
No, that doesn't work. cue `COMPILER_{NATIVE,BYTECODE}_CFLAGS. I think it should be this:

```
native_cflags= "$common_cflags $sharedlib_cflags $native_cflags\
                $PTHREAD_CFLAGS $COMPILER_NATIVE_CFLAGS"
```
as `native_cflags` may already be assigned above with: 
```
native_cflags="$native_cflags $tsan_cflags
```

Reply:
So maybe something like this (untested)?

```
bytecode_cflags="$common_cflags $sharedlib_cflags $bytecode_cflags\
                 $PTHREAD_CFLAGS $COMPILER_BYTECODE_CFLAGS"
native_cflags="$common_cflags $sharedlib_cflags $native_cflags\
                 $PTHREAD_CFLAGS $COMPILER_NATIVE_CFLAGS"
```

The definitions are exactly consistent, and make no assumptions about whether some content has already been pushed into the variable.

Reply:
Thanks, fixed as suggested.